src/plugins/filter/filter-fr.hbs

```diff
@@ -754,3 +754,37 @@
 	...
 &lt;/ul&gt;</code></pre>
 </div>
+
+<h3 id="flt-uicontainer">Move UI to specific container</h3>
+
+<p>Set the <code>source</code> parameter to a jQuery selector that you want to hold the filter's UI. The UI will be added to the top (prepend) of said element.</p>
+
+<div class="row">
+	<div class="col-md-6">
+		<ul class="wb-filter" data-wb-filter='{ "source": "#filterUIContainer" }'>
+			<li>Alberta</li>
+			<li>British Columbia</li>
+			<li>Manitoba</li>
+			<li>New Brunswick</li>
+			<li>Newfoundland &amp; Labrador</li>
+			<li>Northwest Territories</li>
+			<li>Nova Scotia</li>
+			<li>Nunavut</li>
+			<li>Ontario</li>
+			<li>Prince Edward Island</li>
+			<li>Quebec</li>
+			<li>Saskatchewan</li>
+			<li>Yukon</li>
+		</ul>
+	</div>
+	<div class="col-md-6">
+		<div id="filterUIContainer"></div>
+	</div>
```

Comment:
Same comment as English, move the UI before the list being filtered. Otherwise a link should be added after to let the user to go back to the filtered list.